src/main/java/org/zalando/nakadi/service/subscription/state/ClosingState.java

```diff
@@ -157,8 +158,8 @@ private void freePartitions(final Collection<EventTypePartition> keys) {
                     listener.close();
                 } catch (final RuntimeException ex) {
                     exceptionCaught = ex;
-                    TracingService.logErrorInSpan(TracingService.activateSpan(getContext().getCurrentSpan(), false),
-                            String.format("Failed to cancel offsets listener {} {}" , listener, ex.getMessage() ));
+                    TracingService.logErrorInSpan(getContext().getCurrentSpan(),
```

Comment:
user will definitely not need it, it is quite internal exception, which we could track in the logs